setup-gcloud/__tests__/client-util.test.ts

```diff
@@ -23,78 +23,84 @@ import * as util from '../src/client-util';
 const TEST_TIMEOUT_MILLIS = 10000;
 
 describe('queryGcloudSDKRelease tests', () => {
-    it(
-        'Finds matching version linux',
-        async () => {
-            const release: util.IGcloudSDKRelease | null = await util.queryGcloudSDKRelease(
-                'linux',
-                'x86_64',
-                testUtil.TEST_SDK_VERSION,
-            );
-            expect(release).toBeDefined();
-            expect(release!.name).toBe(`google-cloud-sdk-${testUtil.TEST_SDK_VERSION}-linux-x86_64.tar.gz`);
-            expect(release!.version).toBe(testUtil.TEST_SDK_VERSION);
-            expect(release!.url).toEqual(
-                expect.stringContaining(
-                    `https://www.googleapis.com/download/storage/v1/b/cloud-sdk-release/o/google-cloud-sdk-${testUtil.TEST_SDK_VERSION}-linux-x86_64.tar.gz`,
-                ),
-            );
-        },
-        TEST_TIMEOUT_MILLIS,
-    );
+  it(
+    'Finds matching version linux',
+    async () => {
+      const release: util.IGcloudSDKRelease | null = await util.queryGcloudSDKRelease(
```

Comment:
nit: `queryGcloudSDKRelease` -> `queryGcloudSdkRelease` technically not wrong but might help improve readability.